21torr/hosting/3.2/config/routes/hosting.yaml

```diff
@@ -0,0 +1,3 @@
+_import.hosting:
```

Comment:
I think according to current bundle name it should be plain
```suggestion
hosting:
```

Reply:
The keys of import routes are removed, right? Otherwise we might have an issue with name collisions.